### ISSUE TITLE
Support for JSON

### ISSUE THREAD

**Issue by bachgarash** [varbhat/cvtek#3]
Hi. I have used jsonresume ( https://jsonresume.org/getting-started) previously. However my experience with it was not nice, although I assume having JSON as CV and ability to generate it as multiple templates is a good idea. What do you think about it ?

**Comment by varbhat** [comment]
@Bakhtiyar-Garashov support for json is already present in cvtek, thanks to serde :)

**Comment by varbhat** [comment]
@Bakhtiyar-Garashov , yeah. json is a good idea. but, i had implemented the yaml format because yaml was more human writable than json.

**Comment by varbhat** [comment]
Adding support for json should be easy thanks to serde.

kindly take a look here: 

https://github.com/varbhat/cvtek/blob/2996a3ae21b7a78331b567dccf84ecbe68e6a469/src/main.rs#L18
